compiler/qsc_fir/src/fir.rs

```diff
@@ -351,6 +587,36 @@ impl Display for Package {
     }
 }
 
+impl PackageLookup for Package {
+    fn get_block(&self, id: BlockId) -> &Block {
+        self.blocks.get(id).expect("Block not found")
+    }
+
+    fn get_expr(&self, id: ExprId) -> &Expr {
+        self.exprs.get(id).expect("Expression not found")
+    }
+
+    fn get_global(&self, id: LocalItemId) -> Option<Global> {
+        match &self.items.get(id)?.kind {
+            ItemKind::Callable(callable) => Some(Global::Callable(callable)),
+            ItemKind::Namespace(..) => None,
+            ItemKind::Ty(..) => Some(Global::Udt),
+        }
+    }
+
+    fn get_item(&self, id: LocalItemId) -> &Item {
+        self.items.get(id).expect("Item not found")
+    }
```

Comment:
`get_item` is almost the same as `get_global`, but it returns namespaces as well. It's also unused (rather, it's called from the corresponding `PackageStore` method, but _that_ method is unused). Do we need it in the interface, or can we remove it to reduce ambiguity?

Reply:
I think we can remove it. I'll do that.

Reply:
Removed.